dev/com.ibm.ws.jndi.iiop/bnd.bnd

```diff
@@ -47,3 +48,13 @@ instrument.classesExcludes: com/ibm/ws/jndi/iiop/*.class
 	com.ibm.ws.jndi;version=latest,\
 	com.ibm.ws.transport.iiop;version=latest, \
 	com.ibm.ws.org.osgi.annotation.versioning;version=latest
+
+-testpath: \
```

Comment:
Should the copyright be updated?

Reply:
I've assumed it should and updated it accordingly.

Reply:
It should be 2017,2021, not just 2021

Reply:
Of course it should. Thanks.